app/views/conversations/show.html.erb

```diff
@@ -37,7 +37,15 @@
   <div class="request-container">
     <div class="conversation-request-card">
       <div class="conversation-request-card-status">
-        <p id="request-status"><%= @request.status.capitalize %></p>
+        <% if @request.helper_id.present? %>
+          <% if @other_user.photo.attached? %>
```

Comment:
Also, here, instead of having nested if statements you can add it to the if statement as a &&. That way you effectively check if both of those conditions are true are on single line.

Reply:
I nested it because it might be that a user has no photo. 
And I want to check if the request.status is not pending. 
And then depending if extisting or not add the users avatar or the placeholder

Reply:
If `@request.helper_id.present` is false, there can be no photo since there is no user. Adding a second if statement seems less readable than saying `if @request.helper_id.present? && @other_user.photo.attached do x.` Just my personal opinion though.

Reply:
I'm ok with both solutions
But it's not only about readability but the actual functionality. 
If we prefer not to show a placeholder avatar then the "if && condition" makes sense.

Reply:
Ahh, I see what you're saying now. I misread it before. You're right, we need the nested if's for the placeholder to render correctly.